tests/ui/diagnostic_namespace/on_unimplemented_ice.rs

```diff
@@ -1,17 +0,0 @@
-//@ edition:2021
```

Comment:
Github didn't recognize the file being moved, but I gave it a better name. Also, there's no reason for us to be using `#[test]` here -- it's not necessary to reproduce the ICE; all that is necessary is a foreign crate.